koku/api/cloud_accounts/serializers.py

```diff
@@ -25,3 +25,11 @@ class CloudAccountSerializer(serializers.Serializer):
     value = serializers.CharField()
     description = serializers.CharField()
     updated_timestamp = serializers.DateTimeField()
+
+
+class CloudAccountQueryParamsSerializer(serializers.Serializer):
+    """Serializer for the Query params on Cloud Account."""
+
+    limit = serializers.IntegerField(required=False, min_value=1)
+    offset = serializers.IntegerField(required=False, min_value=0)
+    page = serializers.IntegerField(required=False, min_value=1)
```

Comment:
what is page for?

Reply:
The API comes with pagination built in, for example when I add a second test Cloud Account, the page can be used, in combination with limit, to select the first or second cloud account.

Reply:
nevermind, I don't think page is used.

Reply:
I removed it now.